assets/js/dashboard.js

```diff
@@ -87,6 +89,14 @@ let vm = new Vue({
         recipeVersionUrl (recipe, version) {
             return recipe.publicUrl + '/' + version
         },
+        getRepoPackagistUrl (recipe, privatePackagistUrl) {
+            this.packagistUrl = recipe.repo.slug === this.privateSlug && privatePackagistUrl !== '' ? privatePackagistUrl : this.defaultPackagistUrl;
```

Comment:
Doesn't seem right to set a property in a getter? This should probably be sorted out into a computed property. Let me know if you need help with this.